grafana-plugin/playwright.config.ts

```diff
@@ -64,7 +64,7 @@ export default defineConfig({
     // baseURL: 'http://localhost:3000',
 
     trace: 'on',
-    video: 'on',
+    video: 'off',
```

Comment:
question: why not have the `trace` and `video` options set to `on-first-retry`?

Reply:
Hey Michael, thanks for the comment! We discussed that we don't really use videos, just traces, that's why it made sense to turn it off. Regarding setting `on-first-retry` for traces, I think traces are cheap in terms of time and output size and still I guess it might be useful to have a full report with traces even for passed tests